src/CloudGraphsDFG/services/CommonFunctions.jl

```diff
@@ -133,6 +133,10 @@ Build a Cypher-compliant set of properies from a subnode type.
 Note: Individual values are serialized if they are not already.
 If the additional properties is provided, the additional
 properties will be added in verbatim when serializing.
+
+Related
+
+_Neo4jTo???
 """
 function _structToNeo4jProps(inst::Union{User, Robot, Session, PVND, N, APPE, ABDE},
                              addProps::Dict{String, String}=Dict{String, String}();
```

Comment:
HI @GearsAD , could you perhaps update the partner `Neo4j-->Struct` function to also use the convert function as I have done here.  I was not able to find it in the code base :-P.  Whenever you get a chance and if it's relevant please.